### PR TITLE
Autodetect Feature, effectively fixes #4 and #10

### LINKED ISSUE #4: user menu entry
Hey,

Wouldn’t it be better if the menu entry for presentation mode would stand under the "panelStatus" button in the top right corner?

The current problem is: If the laptop battery isn't plugged in, the power menu button is hidden and I can't activate the presentation mode.

Greetings

### PR DESCRIPTION
**Pull request by Mystro256**
After seeing #10 (and originally #4), I wrote a little fix to make the extension automatically check if the battery menu is there, if not it uses the accessibility menu. Also taking into account that the user may have this disabled, I also made it use the usermenu instead if a11y is not available.

A much needed fix for sure :)

### CODE REVIEW

**Comment by RaphaelKimmig** [comment]
line 33 looks like a typo to me?
I'll test the patch as soon as I can and then merge it. 
Thank you very much for sharing your improvements!

**Comment by Mystro256** [comment]
Ih yeah, sorry i was in a rush when I committed it. The whole line 33 is unnecessary, as it was just a line I used in testing the functionality of the changes.

Good eye, I'll mark it.

File: presentationmode@ampad.de/extension.js [hunk 1]
```diff
@@ -22,14 +22,30 @@ let SessionProxy = DBus.makeProxyClass(SessionIface);
...
 function init(extensionMeta) {
+    // To Avoid issues and glitching, this needs to be placed here as
+    // batteryMenu must be constant after initialization to avoid
+    // problems with enable and disable functions
+
+    batteryMenu = Main.panel._statusArea.battery;
+    let temp = Main.panel._statusArea.battery._deviceItems;
+    if(temp == "")
+    {//check for no battery or power device, i.e. no power menu
+        lettemp = new Main.Panel.STANDARD_STATUS_AREA_SHELL_IMPLEMENTATION["a11y"]
```
Comment (Mystro256) [inline presentationmode@ampad.de/extension.js]: Line 33 should be removed :)

**Comment by Mystro256** [comment]
I added some more improvements! :)
I just forgot to remove that line, I'll do that now.

**Comment by Mystro256** [comment]
Also, I did some testing; this seems to work just fine on several laptops and desktops (with and without the a11y menu). The only issue I've seen is that this does not work with "gnome-shell-reflection" (https://github.com/gecos-team/gnome-shell-reflection) unless noa11y is also enabled at the same time (the no-a11y functionality built in is a little broken)

I'm half tempted to fork it, fix it up, and remove that no-a11y functionality, so I can request a fedora package for it. ;)

**Comment by Mystro256** [comment]
This seems broken and have no interest in fixing it

**Pull request closed without merging**